pkg/build/config/version.go

```diff
@@ -81,9 +81,6 @@ func GenerateGrafanaVersion(buildID, grafanaDir string) (string, error) {
 		buildID = shortenBuildID(buildID)
 		verComponents := strings.Split(version, "-")
 		version = verComponents[0]
-		if len(verComponents) > 1 {
```

Comment:
This also cuts things like `beta.1` etc., right? I don't think that's a case that is relevant here IMO, though.

Reply:
@zerok I think we are safe based on the tests here https://github.com/grafana/grafana/blob/main/pkg/build/config/genmetadata_test.go#L25

Reply:
Yes 🙂